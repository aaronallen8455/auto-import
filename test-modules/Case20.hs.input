module Case20 where

main :: IO ()
main = _Just
