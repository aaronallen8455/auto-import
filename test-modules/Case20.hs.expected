module Case20 where

import Control.Lens (_Just)

main :: IO ()
main = _Just
